CRToast/CRToastConfig.m

```diff
@@ -750,7 +762,8 @@ BOOL CRToastAnimationDirectionIsHorizontal(CRToastAnimationDirection animationDi
     return !CRToastAnimationDirectionIsVertical(animationDirection);
 }
 
-static CGFloat kCRCollisionTweak = 0.5;
+//Este valor estaba a 0.5. Lo cambiamos a variable de configuración porque a partir de iOS>=9 hace que detecte la colisión mal y es necesario ponerlo a 0.0 ahora ver kCRToastCollisionTweakKey
```

Comment:
since the rest of the comments are in English, it'd probably be best to translate this to English too 😅

Reply:
👍 i translate it. My change was too nightly and for my own use so i didn't even notice the language of the comment